packages/appium/docs/zh/index.md

```diff
@@ -1,6 +1,21 @@
 ---
-title: Appium文档
+title: Appium文档首页
 ---
+![image](https://github.com/appium/appium/assets/48639567/9dbaf0a0-df6c-4d61-be3e-617653ee1065)
```

Comment:
Not sure if linking to the file through URL is the best approach. Probably better to use the same HTML code from the EN docs (to match the image position and size), just change `src="../en/assets/images/appium-logo-horiz.png"`
In the future we will probably move all images outside the EN directory, so docs for other languages don't need to always check the EN directory.

Reply:
Get it, I will correct it.